DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: fitzRoy
 Title: Easily Scrape and Process AFL Data
-Version: 1.5.0.9000
+Version: 1.5.1
```

Comment:
I've been following the tidyverse convention of only increasing major/minor/patch for releases that go to CRAN. 

Since this is still a development version, you can just increment it to 1.5.0.9001 

https://r-pkgs.org/lifecycle.html#sec-lifecycle-version-number